WebMessageParser.cpp

```diff
@@ -1,44 +1,107 @@
 //
 // Created by Emil on 12/30/18.
 //
-
 #include "WebMessageParser.h"
+#include <iostream>
```

Comment:
nie powinny te includy iść do nagłówkowego? imo nie znam się ale się zastanawiam czy to aby na pewno tutaj